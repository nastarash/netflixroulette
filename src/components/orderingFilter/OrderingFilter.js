import React from 'react';

class OrderingFilter extends React.Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    return <div>Sort by: bla-bla-bla</div>;
  }
}

export default OrderingFilter;
