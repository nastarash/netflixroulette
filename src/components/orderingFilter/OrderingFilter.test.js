import React from 'react';
import renderer from 'react-test-renderer';
import OrderingFilter from './OrderingFilter';

it('render correctly', () => {
  const AppComponent = renderer.create(<OrderingFilter />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});
