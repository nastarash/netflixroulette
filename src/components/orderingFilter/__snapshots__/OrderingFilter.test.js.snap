// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div>
  Sort by: bla-bla-bla
</div>
`;
